example/spec/helpers/ruleUtils.js

```diff
@@ -0,0 +1,28 @@
+const pWaitFor = require('p-wait-for');
+const { listRules } = require('@cumulus/integration-tests/api/rules');
+
+/**
+ * Wait for the defined rule to exist in the rules list.
+ *
+ * @param {string} stackName - Deployment name
+ * @param {Object} ruleQueryParams
+ *   Query params to use for search in rules listing
+ * @returns {Promise}
+ */
+const waitForRuleInList = async (stackName, ruleQueryParams) => pWaitFor(
+  async () => {
+    const resp = await listRules({
+      prefix: stackName,
+      query: ruleQueryParams
+    });
+    return JSON.parse(resp.body).results.length > 0;
```

Comment:
Since the function is called `waitForRuleInList`, returning `undefined` or throwing an exception feels a little more intuitive than `true/false`.

Reply:
I take this comment back, I didn't see that this was for the `pWaitFor` function.

Reply:
`pWaitFor` returns a promise that only resolves if the condition returns `true`

https://github.com/sindresorhus/p-wait-for#pwaitforcondition-options